### PR TITLE
Lightning 1.8.0 release

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Release commit for 1.8.0 with version update and changelog adjustment.

cc @tchaton @rohitgr7 @borda

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]
tumblr_b12eac143b464ec8b2a52531b72ffe2b_e0d0efa4_400

**Review by akihironitta — APPROVED** [review]

File: requirements/app/examples.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-# TODO: as the package is assumed to be independent we shall be using already released versions
-pytorch-lightning @ https://github.com/Lightning-AI/lightning/archive/refs/heads/master.zip
```
Comment (Borda) [inline requirements/app/examples.txt:2]: this dependency on source made it... :confused:

**Review by tchaton — APPROVED** [review]
LGTM

**Review by manskx — APPROVED** [review]

**Review by otaj — APPROVED** [review]

**Review by krshrimali — APPROVED** [review]
🎉

**Review by carmocca — APPROVED** [review]

**Pull request merged**